tests/tests.go

```diff
@@ -4,13 +4,16 @@ import (
 	"context"
 	"fmt"
 	"math"
+	"regexp"
 	"time"
 
 	"k8s.io/apimachinery/pkg/util/rand"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
+	"github.com/k8snetworkplumbingwg/kubemacpool/build/_output/bin/go/src/strconv"
```

Comment:
Where is this coming from ? maybe the IDE you are using is doing this ?

Reply:
yes. will fix